attractions/table/table.svelte

```diff
@@ -0,0 +1,75 @@
+<script>
+  import Label from '../typography/label.svelte';
+
+  /**
+   * @typedef {{
+   *  text: string;
+   *  value: string;
+   *  align?: "start" | "center" | "end";
+   * }} Header
+   * @typedef {{[value: string]: string | number}} Item
+   *
+   * @slot {{ header: Header; item: Item }} item
+   * @slot {{ item: Item }} row
+   */
+
+  /**
+   * The header row of the table. `text` is the text to display, and `value` is used to link the row data.
```

Comment:
I don't remember if our docs support such elaborate markup, but I hope so
```suggestion
   * The labels for the column headers.
   *
   * Each column header is an object with the following fields:
   * - `text` is the text to display;
   * - `value` is the name of the field to take from each object in `items` to put in this column;
   * - `align` is the alignment of the text in the column (`"start" | "center" | "end"`).
```

Reply:
> our docs

This is part of the component's JSDoc, not our `svx` documentation. Did you mean to suggest this there?

Reply:
Oh, right, I forgot that we don't use the JSDocs in the actual documentation. Well, I think it works in both